docs/.custom_wordlist.txt

```diff
@@ -4,20 +4,25 @@ DaemonSet
 DS
 DSS
 GeForce
+gpu
 GPUs
 hostpath
 Hostpath
+HWE
 Initialize
 initialize
 initialized
 initializing
+intel
 io
 ipynb
 Jupyter
 jupyter
 JupyterLab
 kubeconfig
+kubectl
 kubeflownotebookswg
+kustomization
```

Comment:
@mvlassis You need to add "kustomize" as well. The current deployment is broken.

In case you are not aware, run often the following to check for potential errors:
`make spelling`
`make linkcheck`